app/views/setup/welcome.html.slim

```diff
@@ -1,47 +1,67 @@
 h1 Initial CaaSP Configuration
 = form_for :settings, url: setup_path, method: :put do |f|
-  .form-group
-    = f.label :dashboard, "Dashboard location"
-    = f.text_field :dashboard, value: (Pillar.value(pillar: :dashboard) || request.host), class: "form-control", required: true
-  .form-group
-    = f.label :apiserver, "External Kubernetes API server FQDN"
-    = f.text_field :apiserver, value: Pillar.value(pillar: :apiserver), class: "form-control", required: true
-  .form-group
-    = f.label :http_proxy, "HTTP proxy for Docker"
-    = f.url_field :http_proxy, value: Pillar.value(pillar: :http_proxy), class: "form-control"
-  .form-group
-    = f.label :https_proxy, "HTTPS proxy for Docker"
-    = f.url_field :https_proxy, value: Pillar.value(pillar: :https_proxy), class: "form-control"
-  .form-group
-    = f.label :no, "No-proxy settings for Docker"
-    = f.text_field :no_proxy, value: Pillar.value(pillar: :no_proxy), class: "form-control"
-  .form-group
-    = f.label :company_name, "Company Name"
-    = f.text_field :company_name, value: Pillar.value(pillar: :company_name), class: "form-control", required: true
-  .form-group
-    = f.label :company_unit, "Company Unit"
-    = f.text_field :company_unit, value: Pillar.value(pillar: :company_unit), class: "form-control", required: true
-  .form-group
-    = f.label :email, "E-Mail"
-    = f.email_field :email, value: Pillar.value(pillar: :email), class: "form-control", required: true
-  .form-group
-    = f.label :country
-    - if Pillar.value(pillar: :country)
-      = f.select :country, options_for_select(country_codes, Pillar.value(pillar: :country)), {}, class: "form-control"
-    - else
-      = f.select :country, country_codes, { include_blank: true }, class: "form-control"
-  .form-group
-    = f.label :state
-    = f.text_field :state, value: Pillar.value(pillar: :state), class: "form-control", required: true
-  .form-group
-    = f.label :city
-    = f.text_field :city, value: Pillar.value(pillar: :city), class: "form-control", required: true
+  .panel.panel-default
+    .panel-heading Generic settings
+    .panel-body
+      .form-group
+        = f.label :dashboard, "Dashboard location"
+        .input-group
+          = f.text_field :dashboard, value: (Pillar.value(pillar: :dashboard) || request.host), class: "form-control", required: true
+          span class="input-group-addon"
+            a data-toggle="tooltip" data-placement="left" title="Hostname or IP of the node running this web interface"
+              i class='glyphicon glyphicon-info-sign'
+      .form-group
+        = f.label :apiserver, "External Kubernetes API server FQDN"
+        .input-group
+          = f.text_field :apiserver, value: Pillar.value(pillar: :apiserver), class: "form-control", required: true
+          span class="input-group-addon"
+            a data-toggle="tooltip" data-placement="left" title="FQDN pointing to the node acting as Kubernetes master"
```

Comment:
What about some additional context (assuming it fits decently), on the lines of:

` Fully qualified domain name used to reach the cluster from the outside. In a simple, single master deployment this will be the FQDN of the node you are about to select as master. For advanced options refer to the documentation.`